src/test/java/org/checkerframework/specimin/UnusedTypeParameterBoundTest.java

```diff
@@ -0,0 +1,18 @@
+package org.checkerframework.specimin;
+
+import java.io.IOException;
+import org.junit.Test;
+
+/**
+ * This test checks if Specimin can remove unused bounds of type parameters in the declaration of a
+ * class.
```

Comment:
Can you please add a test for generic type bounds that should not be removed, as well. We don't currently have one.

Reply:
(to check that, I searched our `src/test/resources/` directory for "extends", and then observed that none of the results are type variables)